pkg/templates/user-data.template

```diff
@@ -26,7 +26,14 @@ for file in *; do
 		chmod u+x "$file"
 	fi
 done
+
+wget https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
```

Comment:
This section is more or less hard-coded; wanted to get this working and get some input/feedback before making it more robust. The following can be improved:
* use *regional* download link
* the architecture in the download link also supports arm64-- so need to introduce logic to change this value based on $ARCHITECTURE
  * no arm testing right now; should add that too
* config file `cwagent-config.json` _could be_ dynamic, but I think I'll let IQ enforce the name here

Reply:
Will need to support Debian based OSs as well as Redhat ones.

Reply:
good catch! reading docs is hard